test/powershell/Modules/Microsoft.PowerShell.Management/Wait-Process.Tests.ps1

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Wait-Process" {
+
+    BeforeAll {
+        $isNanoServer = [System.Management.Automation.Platform]::IsNanoServer
+        $isIot = [System.Management.Automation.Platform]::IsIoT
+        $isFullWin = $IsWindows -and !$isNanoServer -and !$isIot
+
+        $pingCommandPath = (Get-Command -CommandType Application ping)[0].Definition
+
+        $startProcessArgs = @{
+            FilePath = $pingCommandPath
+            PassThru = $true
+        }
+        if ($isFullWin) {
+            $startProcessArgs.WindowStyle = [System.Diagnostics.ProcessWindowStyle]::Hidden
```

Comment:
Why do we need this for tests?

Reply:
Honestly, I don't know that we do.  I used [Start-Process.Tests.ps1](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1) as a starting point.  Since those tests bothered to specify `WindowStyle`, I figured the `Wait-Process` tests should as well.

Reply:
Is there a verdict on this revision?  If specifying `WindowStyle` isn't needed here, should it be removed from the `Start-Process` tests as well?

Reply:
I hope MSFT team makes a conclusion.
/cc @JamesWTruher @adityapatwardhan

Reply:
I'm guessing the original intent is that when you run these tests locally, processes wouldn't be popping up and take the active window.  It's probably ok to keep or not keep it since most tests are being run in CI now and not locally (where the environment is more variable).

Reply:
That was my guess as well, and I did indeed find it convenient while working on the tests locally.  If it's useful to someone adding more tests later and doesn't noticeably affect CI performance in the meantime, I'd lean towards keeping it in.

Reply:
there is definitely an issue with `WindowStyle` when running in our release tests (we're not _completely_ sure why, but believe it is related to how some of the images are used in AzDevOps pipelines). If it's not needed, I would suggest that you remove it. As it's a Windows only thing anyway, I would rather remove it and live with the mild irritation of local testing (since it's only eliminated on windows)

Reply:
> there is definitely an issue with `WindowStyle` when running in our release tests (we're not _completely_ sure why, but believe it is related to how some of the images are used in AzDevOps pipelines). If it's not needed, I would suggest that you remove it. As it's a Windows only thing anyway, I would rather remove it and live with the mild irritation of local testing (since it's only eliminated on windows)

Weird!  I've removed the code in question.